code/core/src/cli/eslintPlugin.ts

```diff
@@ -7,22 +7,23 @@ import chalk from 'chalk';
 import { readConfig, writeConfig } from '@storybook/core/csf-tools';
 import type { JsPackageManager } from '@storybook/core/common';
 import { paddedLog } from '@storybook/core/common';
+import fs from 'node:fs';
 
 export const SUPPORTED_ESLINT_EXTENSIONS = ['js', 'cjs', 'json'];
 const UNSUPPORTED_ESLINT_EXTENSIONS = ['yaml', 'yml'];
 
 export const findEslintFile = () => {
   const filePrefix = '.eslintrc';
   const unsupportedExtension = UNSUPPORTED_ESLINT_EXTENSIONS.find((ext: string) =>
-    fse.existsSync(`${filePrefix}.${ext}`)
+    fs.existsSync(`${filePrefix}.${ext}`)
   );
```

Comment:
mocking out fs-extra is kind of impossible because it gets prebunded. However, existSync just exist on `fs`. So used that instead to make sure it can get unit tested.

We might want to get rid of `fs-extra` anyway, it is not really necessary anymore with the new Node API

Reply:
Great!, I've been making an effort to import from `'node:fs/promises'` whenever possible.

I actually write a ESM focussed version of `fs-extra` that ONLY contains things that `node:fs/promises` does not:
https://www.npmjs.com/package/@ndelangen/fs-extra-unified